docs/pages/custom-builds/schema.mdx

```diff
@@ -570,6 +570,100 @@ EAS provides a set of built-in reusable functions that you can use in a workflow
 
 > **info** **Tip:** Any function that is built-in and provided by EAS must start with the `eas/` prefix.
 
+#### `eas/maestro_test`
+
+All-in-one function that installs Maestro, prepares a testing environment (iOS Simulator or Android Emulator) and tests the app.
+
+| Input       | Description                                                                                                                                                 |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `flow_path` | **string** Path (or multiple paths, each in a separate line) to [Maestro flows](https://maestro.mobile.dev/getting-started/writing-your-first-flow) to run. |
+
+
+```yaml build-and-test.yml
+build:
+  name: Build and test
+  steps:
+    - eas/build
+    # @info #
+    - eas/maestro_test:
+        inputs:
+          flow_path: |
+            maestro/sign_in.yaml
+            maestro/create_post.yaml
+            maestro/sign_out.yaml
+    # @end #
+```
+
+ Behind the scenes, it uses:
+- [`eas/install_maestro`](#easinstall_maestro) to install Maestro,
+- [`eas/start_android_emulator`](#easstart_android_emulator) to start an Android Emulator if needed,
+- [`eas/start_ios_simulator`](#easstart_ios_simulator) to start an iOS Simulator if needed,
+- custom `run` to install APK/APP to the running Emulator/Simulator,
+- series of `run`s to execute `maestro test` for each of the provided flows,
+- [`eas/upload_artifact`](#easupload_artifact) to upload Maestro test artifacts as build artifact.
```

Comment:
```suggestion
- [`eas/install_maestro`](#easinstall_maestro) to install Maestro
- [`eas/start_android_emulator`](#easstart_android_emulator) to start an Android Emulator if needed
- [`eas/start_ios_simulator`](#easstart_ios_simulator) to start an iOS Simulator if needed
- Custom `run` to install **.apk**  to the running Android Emulator and **.app** to iOS Simulator
- Series of `run` to execute `maestro test` for each of the provided flows
- [`eas/upload_artifact`](#easupload_artifact) to upload Maestro test artifacts as build artifact
```

- We can avoid punctuation from these steps
- In third last step, let's descriptive about the Android and iOS virtual devices and also mention the file formats.